sdks/go/gogradle.lock

```diff
@@ -5,7 +5,7 @@ dependencies:
   build:
   - vcs: "git"
     name: "cloud.google.com/go"
-    commit: "4f6c921ec566a33844f4e7879b31cd8575a6982d"
+    commit: "03869a08dc16b35ad4968e92d34c5a2a2961b205"
```

Comment:
This PR shouldn't require any changes to dependencies.

Reply:
The existing version of google.golang.org/api dependency doesn't have fields _worker_region_ and _worker_zone_. The change to google.golang.org/api version lead to other dependency changes.

Reply:
> The existing version of google.golang.org/api dependency doesn't have fields worker_region and worker_zone.

I see. I was not expecting that because worker_region and worker_zone were added a long time ago. The old Google API library is from Feb 7 2018.

Anyway, we should split the dependency changes and feature implementation into separate commits.

Reply:
It's a pain in the butt to update/add new dependencies to the Go SDK, so it hasn't been kept up. This will change after we're on Go Modules for dependency versioning, which will happen after Schemas are finished implementation in the SDK. (I'm working on it). That would also allow users to use the same deps as the project uses for testing. Right now, this list is functionally for Jenkin's Use Only.

While your proposal to split the changes is generally a good policy, I disagree that it's necessary to split the changes in this instance. There's not nearly as much surface area vs the Python and Java SDKs, and there are no real runners implemented in Go. If the PostCommit test passes, there's little else we do that would detect issues with GRPC or similar.

Also, internal testing at Google is likely using much more up to date versions of all these packages so this just brings things closer to that.

Reply:
> While your proposal to split the changes is generally a good policy, I disagree that it's necessary to split the changes in this instance.

Not necessary, just a matter of personal preference (they can just be separate commits, it's fine if they're merged as part of the same PR).

Reply:
thanks both for the elaboration.

i will merge changes to the dependencies in one PR since it only impacts jenkins